dist/etc/bash_completion.d/elbencho

```diff
@@ -149,6 +149,7 @@ _elbencho_opts()
         --s3otagverify
         --s3randobj
         --s3region
+        --s3sessiontoken
```

Comment:
there is another place a bit further down in the file where this needs to be added. (you can search e.g. for `s3secret` to find the other place.) this other place is needed for the bash completion to knows that the following thing will be a custom string and not another `--option` to complete.